src/Elliptic/Actions/InitializeSystem.hpp

```diff
@@ -0,0 +1,125 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/Mesh.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "NumericalAlgorithms/LinearSolver/Tags.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "ParallelAlgorithms/Initialization/MergeIntoDataBox.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace elliptic {
+namespace Actions {
+
+/*!
+ * \brief Initializes the DataBox tags related to the elliptic system
+ *
+ * The system fields are initially set to zero. The linear solver operand and
```

Comment:
Why not set them initially to signaling nans, to make sure they are not used when not set correctly?

Reply:
They are set to NaN in debug builds and undefined in release builds (that's how the Variables initializer works). As far as the documentation is concerned, it's undefined.